media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,234 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.2-gtk3"
+
+inherit cmake wxwidgets xdg
+
+MY_P="Audacity-${PV}"
+DOC_PV="${PV}"
+DESCRIPTION="Free crossplatform audio editor"
+HOMEPAGE="https://web.audacityteam.org/"
+
+# A header-only thread pool library, without a build system, about 100
+# lines of code.  Probably not worth packaging individually.  Check
+# cmake-proxies/CMakeLists.txt and search for "ThreadPool".
+MY_THREADPOOL_DATE=20140926
+MY_THREADPOOL="https://raw.githubusercontent.com/progschj/ThreadPool/9a42ec1329f259a5f4881a291db1dcb8f2ad9040/ThreadPool.h -> progschj-ThreadPool-${MY_THREADPOOL_DATE}.h"
+SRC_URI="https://github.com/audacity/audacity/archive/${MY_P}.tar.gz
+	doc? ( https://github.com/audacity/audacity-manual/releases/download/v${PV}/audacity-manual-${PV}.tar.gz )
+	telemetry? ( ${MY_THREADPOOL} )
+	audiocom? ( ${MY_THREADPOOL} )"
+S="${WORKDIR}/${PN}-${MY_P}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="alsa audiocom doc ffmpeg +flac id3tag +ladspa +lv2 mad mpg123 ogg
+	opus oss +portmixer sbsms telemetry twolame vamp +vorbis wavpack"
+
+# The testsuite consists of two tests, 50% of which fail.
+RESTRICT="test"
+
+# dev-db/sqlite:3 hard dependency.
+# dev-libs/glib:2, x11-libs/gtk+:3 hard dependency, from
+#   cmake-proxies/cmake-modules/dependencies/wxwidgets.cmake
+# sys-apps/util-linux hard dependency, from cmake-proxies/CMakeLists.txt
+#   for libuuid
+# portmidi became non-optional: building without it results in build
+#   failures, even with some of the Debian patches.  It's probably not
+#   in our best interest to fix that as a patch series.
+# glib, gtk and gdk are all directly relied on in the source, not just
+#   through wxGTK
+
+# These are needed for Sentry and audio.com.
+MY_NETDEPEND="
+	net-misc/curl
+	dev-libs/rapidjson
+"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/expat
+	dev-libs/glib:2
+	media-libs/libsndfile
+	media-libs/libsoundtouch:=
+	media-libs/portaudio[alsa?]
+	media-libs/portmidi
+	media-libs/portsmf:=
+	media-libs/soxr
+	>=media-sound/lame-3.100-r3
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	sys-apps/util-linux
+	sys-libs/zlib
+	alsa? ( media-libs/alsa-lib )
+	ffmpeg? ( media-video/ffmpeg )
+	flac? ( media-libs/flac:=[cxx] )
+	id3tag? ( media-libs/libid3tag:= )
+	lv2? (
+		dev-libs/serd
+		dev-libs/sord
+		>=media-libs/lilv-0.24.6-r2
+		media-libs/lv2
+		media-libs/sratom
+		media-libs/suil
+	)
+	mad? ( >=media-libs/libmad-0.15.1b )
+	mpg123? ( media-sound/mpg123 )
+	ogg? ( media-libs/libogg )
+	opus? ( media-libs/opus )
+	sbsms? ( media-libs/libsbsms )
+	telemetry? ( ${MY_NETDEPEND} )
+	audiocom? ( ${MY_NETDEPEND} )
```

Comment:
This seems out of place too, although I can see how it's relevant to `${MY_NETDEPEND}`. But it's a bit confusing.

Reply:
Right, but I didn't figure a better way of specifying "if either A or B" out